src/OpenTelemetry.ResourceDetectors.Azure/README.md

```diff
@@ -31,3 +31,50 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
                             .AddDetector(new AppServiceResourceDetector()))
                         .Build();
 ```
+
+| Attributes recorded by AppServiceResourceDetector |
+|---------------------------------------------------|
+| azure.app.service.stamp                           |
+| cloud.provider                                    |
+| cloud.platform                                    |
+| cloud.resource_id                                 |
+| cloud.region                                      |
+| deployment.environment                            |
+| host.id                                           |
+| service.instance.id                               |
+| service.name                                      |
+
+
+## VM Resource Detector
+
+Adds resource attributes for the applications running in an Azure virtual machine.
+The following example shows how to add `AzureVMResourceDetector` to
+`TracerProvider` configuration:
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.ResourceDetectors.Azure;
+using OpenTelemetry.Resources;
+
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
It would be good to also add the code snippet showing how to add these for metrics.

Reply:
Why only Metrics? The Metrics API is practically identical to the Trace API.
Should we also add a Log example?